### PR TITLE
added grub themes script

### PR DESCRIPTION
**Pull request by guruswarupa**
# Pull Request

## added grub themes script

- grub bootloader theme script added.

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: tabs/system-setup/tab_data.toml [hunk 1]
```diff
@@ -83,10 +83,14 @@ name = "Remove Snaps"
 description = "This script is designed to remove snap"
 script = "4-remove-snaps.sh"
 
-[[data]]
```
Comment (nnyyxxxx) [inline tabs/system-setup/tab_data.toml:86]:
```
[[data]]
```
You cannot remove this.

File: tabs/system-setup/7-grub-theme.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+. ../common-script.sh
+
+themeinstall(){
+    cd "$HOME" && git clone "https://github.com/ChrisTitusTech/Top-5-Bootloader-Themes"
```
Comment (nnyyxxxx) [inline tabs/system-setup/7-grub-theme.sh:5]:
```suggestion
    cd "$HOME/.local/share" && git clone "https://github.com/ChrisTitusTech/Top-5-Bootloader-Themes"
```

File: tabs/system-setup/7-grub-theme.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+. ../common-script.sh
+
+themeinstall(){
```
Comment (nnyyxxxx) [inline tabs/system-setup/7-grub-theme.sh:4]: ``mkdir -p "$HOME/.local/share"``

File: tabs/system-setup/7-grub-theme.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    cd "$HOME" && git clone "https://github.com/ChrisTitusTech/Top-5-Bootloader-Themes"
+    cd "Top-5-Bootloader-Themes"
```
Comment (nnyyxxxx) [inline tabs/system-setup/7-grub-theme.sh:6]:
```suggestion
    cd "$HOME/.local/share/Top-5-Bootloader-Themes"
```

**Review by nnyyxxxx — CHANGES_REQUESTED** [review]

File: tabs/system-setup/tab_data.toml (hunk 1 above)
Comment (guruswarupa) [reply tabs/system-setup/tab_data.toml]: again some issue when i pull the fork ig. i did not touch other part of the script. yet has modification. my bad

File: tabs/system-setup/7-grub-theme.sh (hunk 4 above)
Comment (nnyyxxxx) [reply tabs/system-setup/7-grub-theme.sh:6]: > this isn't needed. we are cd ing to it and cloning. again why pass entire path when relative path can be used?

It is needed, passing the entire path makes errors that could occur nonexistent.

**Review by ChrisTitusTech — APPROVED** [review]

File: tabs/system-setup/7-grub-theme.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+. ../common-script.sh
+
+themeinstall(){
+    mkdir -p "$HOME/.local/share"
+    cd "$HOME/.local/share" && git clone "https://github.com/ChrisTitusTech/Top-5-Bootloader-Themes"
+    cd "$HOME/.local/share/Top-5-Bootloader-Themes"
+    $ESCALATION_TOOL ./install.sh
```
Comment (adamperkowski) [inline tabs/system-setup/7-grub-theme.sh:8]:
```suggestion
    "$ESCALATION_TOOL" ./install.sh
```

**Pull request merged**